src/main/java/core/basesyntax/service/ReadWriteOperations.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import java.io.File;
+import java.util.List;
+
+public interface ReadWriteOperations {
```

Comment:
it is different operations. It must be in different services

Reply:
Not fixed 
5 services for this task will be enough:
RiderService - read data from the file,
WriterService - write data to file,
ReportMakerService - prepare string report,
TransactionParserService - parse every single row from the file to create a list of FruitTransaction
FruitShopService - process the whole list of FruitTransaction using strategy